src/lib/constants.js

```diff
@@ -1,9 +1,13 @@
 // @flow
 
+export const DEFAULT_PORT = '4873';
+export const DEFAULT_DOMAIN = 'localhost';
+
 export const HEADERS = {
   JSON: 'application/json',
   JSON_CHARSET: 'application/json; charset=utf-8',
-  OCTET_STREAM: 'application/octet-stream',
+  OCTET_STREAM: 'application/octet-stream; charset=utf-8',
+  TEXT_CHARSET: 'text/plain; charset=utf-8',
```

Comment:
we can have a constant for `text/html` as well. 

In UI:
<img width="566" alt="screen shot 2018-06-28 at 21 44 14" src="https://user-images.githubusercontent.com/6918450/42058930-2d396408-7b22-11e8-86de-ed972a7ae83c.png">

Reply:
I think all `text/html` should be utf-8. What do you think?

Reply:
Not sure about this.